bot/glossary_command.go

```diff
@@ -0,0 +1,157 @@
+package bot
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/quintilesims/slackbot/db"
+	"github.com/quintilesims/slackbot/models"
+	glob "github.com/ryanuber/go-glob"
+	"github.com/urfave/cli"
+)
+
+// NewGlossaryCommand returns a cli.Command that manages !glossary
+func NewGlossaryCommand(store db.Store, w io.Writer) cli.Command {
+	return cli.Command{
+		Name:  "!glossary",
+		Usage: "manage the glossary",
+		Subcommands: []cli.Command{
+			{
+				Name:      "add",
+				Aliases:   []string{"a"},
```

Comment:
I'm kind of against adding aliases in this manner; it's just more codepaths to test w/o much of a benefit IMO (also, you can always use the `!alias` command)